rust/common-wasi/src/lib.rs

```diff
@@ -0,0 +1,411 @@
+//! # Wasmtime's WASI Implementation
+//!
+//! This crate provides a Wasmtime host implementation of WASI 0.2 (aka WASIp2
```

Comment:
Update this with the fork-diff information, and point to upstream. Preview1 feature was removed.